src/generic_core/globals.ts

```diff
@@ -56,6 +56,7 @@ export var settings :uproxy_core_api.GlobalSettings = {
   proxyBypass: DEFAULT_PROXY_BYPASS.slice(0),
   enforceProxyServerValidity: false,
   validProxyServers: [],
+  promo: null,
```

Comment:
@trevj Happy to add an explanatory comment. @jpevarnek If we initialize it to the promo code, then wouldn't that result in unconditionally applying the promo for all users, instead of just users that installed from the promo landing page?